ext/js/data/anki-note-builder.js

```diff
@@ -103,7 +103,7 @@ export class AnkiNoteBuilder {
             if (internalFieldName === '{url}') {
                 const urlPrefix = '<a href="';
                 const url = value.replace(urlPrefix, '');
-                if (!['http:', 'https:', 'ws:', 'wss:', 'ftp:', 'data:', 'file:'].includes(new URL(url).protocol)) {
+                if (window.location.protocol === new URL(url).protocol) {
```

Comment:
I don't think this is quite right, as `window.location.protocol` can technically be `http(s)` if the code was to execute in a content script (though I guess this code wouldn't execute in a content script, it still doesn't feel right). You instead want `new URL(url).protocol === new URL(import.meta.url).protocol` I believe, as `import.meta.url` will always be the extension URL.

Reply:
Oh ok, thanks. I'll fix this then.